src/main/java/mate/academy/lib/Injector.java

```diff
@@ -57,59 +57,6 @@ public Object getInstance(Class<?> certainInterface) {
         return newInstanceOfClass;
```

Comment:
Are you sure that these changes in necessary?

Reply:
> Are you sure that these changes in necessary?

Had to run "mvn clean package"